core/ir/aarchxx/instr_create_aarchxx.h

```diff
@@ -0,0 +1,51 @@
+/* **********************************************************
+ * Copyright (c) 2011-2020 Google, Inc. All rights reserved.
+ * **********************************************************/
+
+#ifndef INSTR_CREATE_AARCHXX_H
+#define INSTR_CREATE_AARCHXX_H 1
+
+#include "../instr_create_shared.h"
+#include "instr.h"
+
+/* DR_API EXPORT TOFILE dr_ir_macros_aarchxx.h */
+/* DR_API EXPORT BEGIN */
+
+/**
+ * @file dr_ir_macros_aarchxx.h
```

Comment:
This new file "dr_ir_macros_aarchxx.h" is not included anywhere in the exported headers, so this change will break external clients (and our internal tests of the exported headers must not have any clients using DMB).  This file needs a #include inside core/ir/instr_create_shared.h by the other includes of dr_ir_macros_*.